packages/pn-personafisica-webapp/src/components/Contacts/CourtesyContactItem.tsx

```diff
@@ -66,25 +62,30 @@ const CourtesyContactItem = ({ type, value, blockDelete }: Props) => {
 
   const formik = useFormik({
     initialValues: {
-      [type]: '',
+      [type]: value ?? '',
     },
+    enableReinitialize: true,
     validateOnMount: true,
     validationSchema:
-      type === CourtesyFieldType.EMAIL ? emailValidationSchema : phoneValidationSchema,
+      type === CourtesyChannelType.EMAIL ? emailValidationSchema : phoneValidationSchema,
     onSubmit: () => {
       const contactValue =
-        type === CourtesyFieldType.EMAIL
+        type === CourtesyChannelType.EMAIL
           ? formik.values[type]
           : internationalPhonePrefix + formik.values[type];
-      initValidation(digitalDomicileType, contactValue, 'default');
+      if (value) {
+        digitalElemRef.current.editContact();
+      } else {
+        initValidation(digitalDomicileType, contactValue, 'default');
+      }
     },
   });
 
   const handleChangeTouched = async (
     event: ChangeEvent<HTMLTextAreaElement | HTMLInputElement>
   ) => {
     formik.handleChange(event);
-    await handleTouched(event.target.id, true);
+    await handleTouched(event.target.id.toUpperCase(), true);
```

Comment:
why the upperCase?

Reply:
because the name of input is `EMAIL` or `SMS` and the id is `email` or `sms`. So, I have to transform the string to upper cas

Reply:
if we use the variable contactType = type.toLowerCase(), this becomes useless